Swiftplot_Docker_setup.md

```diff
@@ -0,0 +1,54 @@
+# Setting up docker container for SwiftPlot and Swift-Jupyter
+
+To use Swiftplot in a notebook, it requires setting up of swift-jupyter environment, either in Ubuntu or Docker. While either of the approaches are not very difficult, Docker approach provides a consistent way to setup environment. In this document, I will highlight steps needed to setup docker container for Swiftplot.
```

Comment:
`This document highlights the steps needed to setup docker container for Swiftplot.`
Documentations shouldn't be in 1st person, preferably.

Reply:
@KarthikRIyer  - It is a very good point, especially setup documentation!! I will make changes in related areas.